features/steps/test_post_endpoint.rb

```diff
@@ -8,8 +8,9 @@ class Spinach::Features::TestPostEndpoint < Spinach::FeatureSteps
   end
 
   step "I create an Item" do
+
     options = {
-      body: {name: "Foo", quality: 4, sellIn: 4},
+      body: {name: "Foo", quality: 4, sellIn: 4, description: "FooDescription"},
```

Comment:
Especially for acceptance tests, I often find it helpful to have the test data be more meaningful. i.e. I would use "real" information in these steps.

For instance, based on what I'm looking at, the following questions spring to mind:
 * Does the `description` have to be PascalCase?
 * Who is the `description` meant for?

I think these sorts of questions could be alleviated by improving the test data.